rfd/0026/README.md

```diff
@@ -1274,38 +1289,36 @@ an appropriate error message when used on shared volumes zones.
 Machines acting as shared volumes' storage zones will have the value `nfsserver`
 for their `smartdc_role` property. To know of this new `smartdc_role` value is used by CloudAPI to prevent users from performing operations on these storage VMs, refer to
 
-#### New internal `required_nfs_volumes` property on VM objects
+#### New `volumes` property on VM objects
 
 A VM object that represents a Docker container mounting a shared volumes will
 store a reference to that volume in a new _indexable_ property named
-`required_nfs_volumes`.
+`volumes`.
 
 This will allow VOLAPI to perform an indexed search on VMAPI to determine what
 uses a given volume when handling operations that depends on a volume's usage,
 e.g a `DeleteVolume` request.
 
-This property is _internal_ and is not exposed to VMAPI and CloudAPI users. It
-is used by VMAPI's `ListVms` endpoint to implement support for its new
-[`mounting_volume` input
-parameter](#new-mounting_volume-parameter-for-the-listvms-endpoint).
+This property will also be exposed both at VMAPI and CloudAPI where it can be
+passed in as part of a VM payload to mount specified volumes on VM creation,
+and also can be listed via the ListVms and ListMachines endpoints.
 
-#### New `mounting_volume` parameter for the `ListVms` endpoint
+#### New `volumes` parameter for the `ListVms` endpoint
```

Comment:
Should we allow passing multiple volumes UUIDs (separated by a comma?), and if so, would that translate into a "and" or "or" filter?